src/main/java/carpettisaddition/mixins/carpet/tweaks/command/playerActionEnhanced/EntityPlayerActionPackActionMixin.java

```diff
@@ -90,15 +92,24 @@ private void perTickMultiplier(EntityPlayerActionPack actionPack, EntityPlayerAc
 	{
 		if (this.perTick != null)
 		{
+			ServerPlayerEntity player = ((EntityPlayerActionPackAccessor)actionPack).getPlayer();
+			IServerPlayerEntity iplayer = (IServerPlayerEntity)(Object)player;
+			EntityPlayerActionPackActionTypeAccessor typeAccessor = ((EntityPlayerActionPackActionTypeAccessor)(Object)type);
+			EntityPlayerActionPack.Action self = (EntityPlayerActionPack.Action)(Object)this;
+			IEntityPlayerActionPackActionTypeUse actionTypeUSE = (IEntityPlayerActionPackActionTypeUse)(Object)EntityPlayerActionPack.ActionType.USE;
```

Comment:
Does it only support "USE" action pack? imo it's much better if all action packs support interpolation
 at the same time, and them will have the same behavior

Reply:
added to ATTACK as well.

Reply:
You can use the global flag suggestion described in 2. of https://github.com/TISUnion/Carpet-TIS-Addition/pull/119#issuecomment-1762592582, so you don't need to mixin into every action implemetation that invokes `EntityPlayerActionPack#getTarget`